### DISCUSSION TITLE
Adjustments

### DISCUSSION THREAD

**Discussion by IVG-DR** [kellerza/sunsynk#103] [Q&A] [answered]
I have an installed Home Assistant version and the addon. Everything works at the moment.

How can I adjust the configuration for the addon in the Home Assistant so that I can read the 3-phase inverter?

With which tool can I adjust something here?

Thanks!

**Comment by kellerza** [comment]
Its not that simple, there is an open issue where people discuss this

**Reply by IVG-DR** [reply]
OK, I am not a developer myself, but I know the system and its function. I just need to know how to adjust the sensors to the modbus register with an installed addon.

Thanks

**Reply by kellerza** [reply, marked as answer]
Try the new docs & MULTI addon

 https://kellerza.github.io/sunsynk/reference/multi-options
